Source/WebKit/UIProcess/Cocoa/WebProcessPoolCocoa.mm

```diff
@@ -840,6 +877,9 @@ static void logProcessPoolState(const WebProcessPool& pool)
 #endif // !PLATFORM(IOS_FAMILY)
 
 #if PLATFORM(IOS_FAMILY)
+    auto notificationName = adoptNS([[NSString alloc] initWithCString:kGSEventHardwareKeyboardAvailabilityChangedNotification encoding:NSUTF8StringEncoding]);
+    addCFNotificationObserver(hardwareKeyboardAvailabilityChangedCallback, (__bridge CFStringRef)notificationName.get(), CFNotificationCenterGetDarwinNotifyCenter());
```

Comment:
Curious if we could use `CFSTR(kGSEventHardwareKeyboardAvailabilityChangedNotification)` here

Reply:
Ah, good point, will look into that!

Reply:
I am getting a compile error with `CFSTR(kGSEventHardwareKeyboardAvailabilityChangedNotification`.